scripts/fpga/xilinx_vivado_synthesize.tcl

```diff
@@ -13,25 +13,25 @@ create_project  board_specific_top ./gui_prj           \
                -force                                  \
```

Comment:
We need to add files from basics-graphics-music/peripherals here. labs/common only for those modules that are used in user lab_top